tests/test_calculation.py

```diff
@@ -35,6 +36,10 @@ def test_calculation_basic(firecrest_computer: orm.Computer):
     builder = code.get_builder()
     builder.x = orm.Int(1)
     builder.y = orm.Int(2)
+    builder.metadata.options.account = firecrest_config.builder_metadata_options_account
+    builder.metadata.options.custom_scheduler_commands = (
+        "#SBATCH --constraint=mc\n#SBATCH --mem=10K"
```

Comment:
what is this constraint mc?

Reply:
Thanks for the clarification, maybe you can link this here and write a comment that we need this to test against the real server of cscs. If the docker image works again these might change right?

Reply:
This line of the code doesn't need to change in that case. Only in `.firecrest-demo-config.json` one can leave the field empty. So I think it's alright.